flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -411,7 +411,10 @@ public enum JobStoreType {
                     .withDescription("The timeout in milliseconds for a idle slot in Slot Pool.");
 
     /** Config parameter determining the scheduler implementation. */
-    @Documentation.ExcludeFromDocumentation("SchedulerNG is still in development.")
+    @Documentation.Section({
+        Documentation.Sections.EXPERT_SCHEDULING,
+        Documentation.Sections.ALL_JOB_MANAGER
+    })
     public static final ConfigOption<SchedulerType> SCHEDULER =
```

Comment:
I prefer to rename `Ng` to `Default`. I think `Ng` is a legacy name and it may confuse users now.
However, we should still accept the value `Ng` and treat it as `Default` to make the change compatible. But we do not need to expose that to users.
We also need to improve the description of acceptable values to be more explanatory.